cleverhans/attacks_tf.py

```diff
@@ -519,6 +523,7 @@ def __init__(self, sess, model, batch_size, confidence,
         self.clip_min = clip_min
         self.clip_max = clip_max
         self.model = model
+        self.extension = extension
```

Comment:
Add what extension does to the doc string.

Reply:
Done